kubeluigi/__init__.py

```diff
@@ -102,6 +102,8 @@ def run(self):
         self.__logger.info("Submitting Kubernetes Job: " + self.uu_name)
         try:
             run_and_track_job(self.kubernetes_client, job)
+        except:
+            logger.warning("Luigi has failed to submit the job, starting cleaning")
```

Comment:
Don't we want to see the exception here?

Reply:
I could do, the clean function below also prints it though, didn't know if we wanted to double up?

Reply:
ah ok cool